core/src/main/java/zm/gov/moh/core/repository/database/entity/derived/Client.java

```diff
@@ -1,14 +1,23 @@
 package zm.gov.moh.core.repository.database.entity.derived;
-
 import androidx.room.*;
 
 import com.squareup.moshi.Json;
 
 import org.threeten.bp.LocalDateTime;
 
+import androidx.room.Entity;
+
 @Entity
 public class Client {
 
+    public long patient_id;
+    public String identifier;
+    public String given_name;
+    public String family_name;
+    public String gender;
+    public LocalDateTime birthdate;
+
+
     @ColumnInfo(name = "patient_id")
     @Json(name = "patient_id")
     private long patientId;
```

Comment:
@csamiselo `patient_id` and `patientId` are the same. Remove all public member variables when resolving conflicts. @kaykabwe refactored the entities, seek consultation from him. Build is failing.

Reply:
Noted will do